tests/PsrLogAdapterTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace MongoDB\Tests;
+
+use MongoDB\Driver\Logging\Logger as DriverLogger;
+use MongoDB\PsrLogAdapter;
+use PHPUnit\Framework\TestCase as BaseTestCase;
+use Psr\Log\AbstractLogger;
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+
+use function func_get_args;
+use function MongoDB\Driver\Logging\log;
+
+class PsrLogAdapterTest extends BaseTestCase
+{
+    private PsrLogAdapter $adapter;
```

Comment:
Unused. And that's good.

Reply:
Thanks for reminding me. This is now removed.